### PR TITLE
Add the information about the Source

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've added the information about the Source to the `readme.md`.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-clickhouse/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,26 @@ connector, as long as the user has proper access to those tables.
 
 Creating a Destination connector will fail if the table does not exist or if the user does not have permission to work
 with the specified table.
+
+## Source
+
+The ClickHouse Source allows you to move data from the ClickHouse table to any Conduit Destination. When the connector
```
Comment (maksenius) [inline README.md:55]: `to any Conduit Destination ` - unfortunately it is can be not true) . I don't about do we need write this assertion

File: README.md [hunk 2]
```diff
@@ -49,3 +49,26 @@ connector, as long as the user has proper access to those tables.
...
+starts, all the data in the table is transferred to the Destination. After all the data is transferred, the connector
+will transfer the newly added data.
+
+**Important**: update and delete operations are not supported.
+
+### Position
+
+The position contains the last processed element value.
```
Comment (maksenius) [inline README.md:63]: please add example of postilion

File: README.md [hunk 3]
```diff
@@ -49,3 +49,26 @@ connector, as long as the user has proper access to those tables.
 
 Creating a Destination connector will fail if the table does not exist or if the user does not have permission to work
 with the specified table.
+
+## Source
+
+The ClickHouse Source allows you to move data from the ClickHouse table to any Conduit Destination. When the connector
+starts, all the data in the table is transferred to the Destination. After all the data is transferred, the connector
+will transfer the newly added data.
+
```
Comment (maksenius) [inline README.md:58]: Maybe lets just add a short info about snapshot and cdc iterators?

File: README.md (hunk 1 above)
Comment (voscob) [reply README.md:55]: Should I use `some` instead of `any`? Or do you mean I should remove this sentence entirely?

File: README.md (hunk 2 above)
Comment (voscob) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (voscob) [reply README.md:58]: We do not have these iterators in their usual sense. 
So I tried to describe the Source logic without dividing it into iterators.

File: README.md (hunk 1 above)
Comment (maksenius) [reply README.md:55]: maybe just remove word any it is will be looks like:
The ClickHouse Source allows you to move data from the ClickHouse table to Conduit Destination connectors

File: README.md (hunk 3 above)
Comment (maksenius) [reply README.md:58]: But I don't see this description

Comment (voscob) [reply README.md:58]: It is below :)
But I got it.
I'll try to add more information.

File: README.md (hunk 1 above)
Comment (voscob) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (maksenius) [reply README.md:59]: As I remember you use snapshot max value logic we can add few words about it

Comment (voscob) [reply README.md:59]: I've added the info about the iterator.

**Review by maksenius — APPROVED** [review]

**Pull request merged**